src/components/Dropdown/Dropdown.tsx

```diff
@@ -17,13 +16,11 @@ const Dropdown = () => {
   return (
     <div className={styles.dropdown}>
       <ul className={styles.dropdownList}>
-        <Link href={USER_PROFILE_URL}>
+        <a href={USER_PROFILE_URL} className={styles.dropdownLink}>
```

Comment:
cant this be done by link tag?

Reply:
Next 13 gives an error on using a tag inside the link tag but the link tag by itself is not an issue.

Example to use classNames on link tag: https://github.com/vercel/next.js/blob/9a82bba86ed2fba02c651983c78eac744a4d75d3/examples/active-class-name/components/ActiveLink.tsx#L52

Reply:
In case you are doing this so you could place `li` inside `a` you could use this:
https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag